io.openems.edge.ess.sinexcel/src/io/openems/edge/ess/sinexcel/EssSinexcel.java

```diff
@@ -287,13 +313,17 @@ public void islandingOff() throws OpenemsNamedException {
 
 	public void doHandlingSlowFloatVoltage() throws OpenemsNamedException {
 
-		// this.SLOW_CHARGE_VOLTAGE = this.FLOAT_CHARGE_VOLTAGE = (int) (numberOfSlaves
-		// * MAX_CELL_VOLT * NO_OF_CELLS);
+		// this.SLOW_CHARGE_VOLTAGE = this.FLOAT_CHARGE_VOLTAGE = (int) (numberOfSlaves * MAX_CELL_VOLT * NO_OF_CELLS);
+		 //System.out.println(this.SLOW_CHARGE_VOLTAGE);
+		
+		//System.out.println("SLOW_CHARGE_VOLTAGE : " +this.SLOW_CHARGE_VOLTAGE + "FLOAT_CHARGE_VOLTAGE : " +  this.FLOAT_CHARGE_VOLTAGE );
+		
+		System.out.println("Upper voltake : " + this.channel(SinexcelChannelId.UPPER_VOLTAGE_LIMIT).value().asStringWithoutUnit());
```

Comment:
Please remove debug system.out.println or replace them by Logger (`this.logInfo`). Also please remove unnecessary comments if you won't need them anymore (I am aware you are still in development - so only if they are really unnecessary)